units/src/amount/unsigned.rs

```diff
@@ -231,22 +231,34 @@ impl Amount {
     /// Returns [`None`] if overflow occurred.
     pub fn checked_div(self, rhs: u64) -> Option<Amount> { self.0.checked_div(rhs).map(Amount) }
 
-    /// Checked weight division.
+    /// Checked weight ceiling division.
     ///
     /// Be aware that integer division loses the remainder if no exact division
     /// can be made.  This method rounds up ensuring the transaction fee-rate is
-    /// sufficient. If you wish to round down use `amount / weight`.
+    /// sufficient.  See also [`Amount::checked_div_by_weight_floor`].
     ///
     /// Returns [`None`] if overflow occurred.
     #[cfg(feature = "alloc")]
-    pub fn checked_div_by_weight(self, rhs: Weight) -> Option<FeeRate> {
-        let sats = self.0.checked_mul(1000)?;
+    pub fn checked_div_by_weight_ceil(self, rhs: Weight) -> Option<FeeRate> {
```

Comment:
The PR title says `checked_mul` but this is `checked_div` that has been changed

Reply:
Oops, yes.